sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -4094,14 +4094,14 @@ _ZNK4sycl3_V16device9getNativeEv
 _ZNK4sycl3_V16kernel11get_backendEv
 _ZNK4sycl3_V16kernel11get_contextEv
 _ZNK4sycl3_V16kernel13getNativeImplEv
-_ZNK4sycl3_V16kernel16get_backend_infoINS0_4info6device15backend_versionEEENS0_6detail20is_backend_info_descIT_E11return_typeEv
-_ZNK4sycl3_V16kernel16get_backend_infoINS0_4info6device7versionEEENS0_6detail20is_backend_info_descIT_E11return_typeEv
-_ZNK4sycl3_V16kernel16get_backend_infoINS0_4info8platform7versionEEENS0_6detail20is_backend_info_descIT_E11return_typeEv
```

Comment:
These changes are unrelated to the PR, can you please revert them to avoid confusion.

Reply:
This change is automatically done by the script, and it appears in all my PRs.
So, switching the order is the right result according to the tool, and probably is not going to cause this issue again.
But, I just followed your suggestion for this PR.
I believe someone has to commit this order to avoid this issue again.